apps/dashboard/src/utils/telemetry.ts

```diff
@@ -47,4 +47,5 @@ export enum TelemetryEvent {
   TEMPLATE_WORKFLOW_CLICK = 'Template Workflow Click',
   ENVIRONMENTS_PAGE_VIEWED = 'Environments Page Viewed',
   CREATE_ENVIRONMENT_CLICK = 'Create Environment Click',
+  UPGRADE_TO_BUSINESS_TIER_CLICK = 'Upgrade to Business Tier Click',
```

Comment:
is this event used?

Reply:
Yep, on the CTA of the business tier click